test/src/tester.c

```diff
@@ -2096,7 +2101,11 @@ int main(int argc, const char * argv[]) {
 //    igs_stop();
 //    igs_stop();
 
-    igs_start_with_device(networkDevice, port);
+    if (broker) {
+        brokerCommand(broker);
```

Comment:
brokerCommand is not part of the PR and it would be best to add all the broker-related code here directly.